include/boost/math/special_functions/jacobi_theta.hpp

```diff
@@ -0,0 +1,646 @@
+// Jacobi theta functions
+// Copyright Evan Miller 2020
```

Comment:
Make sure to add the Boost Licence under the copyright statement.

Reply:
Will do.